switchio/loop.py

```diff
@@ -51,15 +52,86 @@ def handle_result(task, log, model):
         log.exception("{} failed with:".format(task))
 
 
-class AsyncIOEventLoop(EventLoop):
-    """Re-implementation of the event loop with ``asyncio`` and coroutines
-    support.
-    """
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+class EventLoop(object):
+    '''Processes ESL events using a background (thread) ``asyncio`` event loop
+    and one ``aioesl`` connection.
```

Comment:
You sure you need a separate loop for this? Or that left for investigation after dropping py2?

Reply:
Not sure what you mean. 
Is it the `EventLoop` name that's got you wondering?
This is a proactor / scheduler but also one big loop which processes events.